src/lib/commands/responsive-table.ts

```diff
@@ -50,10 +50,9 @@ function generateHeaderColors(length: number): string[] {
 
 const terminalColumns = process.stdout.columns ?? 100;
 
-export interface ResponsiveTableOptions<
-	AllColumns extends string,
-	MandatoryColumns extends NoInfer<AllColumns> = AllColumns,
-> {
+export const kSkipColumn = Symbol.for('@apify/cli:responsive-table:skip-column');
```

Comment:
why the `k` prefix?

Reply:
Honestly? 🤷‍♂️, I saw it used before in node internals for private symbols and it just kinda stuck. I can drop it for sure

Reply:
then let's mark it as `@internal` at least? i still find this rather confusing (and not consistent, I am not aware of a single symbol following this rule in all our repos), but its not a big deal